site/ja/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contribution Guideline
+
+## Communities
+
+* Slack
+  * Slack の #doc_translation チャンネルで議論をしています
+  * [TensorFlow User Group の公式ページ](https://tfug.jp/)から参加可能です
+* Google Groups
+  * [docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)
+  * [docs-ja@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ja)
+
+## Directory Structure
+
+ドキュメントのディレクトリ構成は次のようになっています:
+
+```
+site
+├── en
+│   ├── ...
+│   ├── ...     
+│   └── ...
+├── ja
+│   ├── ...
+│   ├── ...     
+│   └── ...
+├── ...
+└── ...
+```
+
+`site/en` 以下にあるけれど `site/ja` 以下に無いものは、まだ翻訳が済んでいないドキュメントです。
+それらを翻訳して `site/en` 以下と同じ構成で `site/ja` 以下に配置して pull request を送ってください。
+
+## Pull Request Title
```

Comment:
Use sentence-case for titles and headers (same as developer docs), so "Pull request title"
Here and below

Reply:
Thanks. I fixed it.